### PR TITLE
Refs #25489 - Update fog-vsphere gem to 2.5.0

### PR DESCRIPTION
**Pull request by chris1984**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by chris1984** [comment]
@tbrisker can I get a review please

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 %global scl_ruby_bin /usr/bin/%{?scl:%{scl_prefix}}ruby
 %global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
 
-%global release 3
+%global release 4
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec]: Generally we split these into separate commits.

File: packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec [hunk 2]
```diff
@@ -88,6 +88,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/tests
 
 %changelog
+* Tue Nov 16 2018 Chris Roberts <chrobert@redhat.com> - 2.5.0-1
```
Comment (ekohl) [inline packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec]: This doesn't exist. Note that we have a `bump_rpm.sh` script which does the bumping for you.

**Comment by chris1984** [comment]
New PR https://github.com/theforeman/foreman-packaging/pull/3214
